docker/single_image_testnet/Dockerfile

```diff
@@ -17,10 +17,9 @@ EXPOSE 26657 26659
 RUN ./gen_node_configs.sh
 
 # Update persistent peers list for at least one node
-RUN NODE_0_ID=$(cheqd-noded tendermint show-node-id --home node_configs/node0); \
-    NODE_1_ID=$(cheqd-noded tendermint show-node-id --home node_configs/node1); \
-    PERSISTENT_PEERS="$NODE_0_ID@127.0.0.1:26656,$NODE_1_ID@127.0.0.1:26658"; \
-    sed -i $extension "s/persistent_peers = \"\"/persistent_peers = \"$PERSISTENT_PEERS\"/g" node_configs/node0/config/config.toml
+RUN NODE_0_ID=$(cheqd-noded tendermint show-node-id); \
+    PERSISTENT_PEERS="$NODE_0_ID@127.0.0.1:26656"; \
+    sed -i $extension "s/persistent_peers = \"\"/persistent_peers = \"$PERSISTENT_PEERS\"/g" .cheqdnode/config/config.toml
```

Comment:
And this step expects that there is a directory in the WORKDIR, which is inherited from `cheqd-node` image

Reply:
This change is because previously we had several nodes inside the mage and their homes were `node_configs/node0`, `node_configs/node1`, and so on. Now there is only one node with the home in `.cheqdnode`.